### PR TITLE
Update pkg descriptions for Microsoft.Azure.EventHubs* to include pointer to new packages

### PR DESCRIPTION
**Pull request by ramya-rao-a**
This PR updates the package descriptions for the below two packages to include pointer to new packages and the migration guide
- Microsoft.Azure.EventHubs
- Microsoft.Azure.EventHubs.Processor

As discussed offline, this is in an attempt to guide users to the new package where there is active development while the above packages continue to get critical bug fixes

### CODE REVIEW

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>This is the legacy Azure Event Hubs .NET Standard Event Processor Host library. To view releases for the current generation of the Azure client libraries for .NET, please see: https://aka.ms/azsdk/releases.  For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>4.4.0-beta.1</Version>
+    <Description>Please note, a newer package Azure.Messaging.EventHubs.Processor is available as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```
Comment (jsquire) [inline sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj:3]:
```suggestion
    <Description>Please be aware that a newer package, Azure.Messaging.EventHubs.Processor, was released in February 2020.  While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

File: sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>This is the legacy Azure Event Hubs .NET Standard client library. To view releases for the current generation of the Azure client libraries for .NET, please see: https://aka.ms/azsdk/releases.  For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>4.4.0-beta.1</Version>
+    <Description>Please note, a newer package Azure.Messaging.EventHubs is available as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```
Comment (jsquire) [inline sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj:3]:
```suggestion
```suggestion
    <Description>Please be aware that a newer package, Azure.Messaging.EventHubs, was released in February 2020.  While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj (hunk 1 above)
Comment (jsquire) [reply sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj:3]: In my opinion, the `for new development` is a significant distinction.  While we'd like to see migration in existing applications, I don't believe there is forcing function for it nor that there is enough of a value proposition to justify the efforts for many applications with an existing investment in the legacy packages.

**Review by jsquire — APPROVED** [review]
LGTM;  I'd strongly recommend that we wait on @serkantkaraca's blessing before merging.

**Comment by jsquire** [comment]
Was there any reason why the README for Service Bus was annotated, but the ones for the Event Hubs packages were not?

**Comment by ramya-rao-a** [comment]
> Was there any reason why the README for Service Bus was annotated, but the ones for the Event Hubs packages were not?

Probably because I am operating on very less sleep and in between meetings :)

I'll fix that

**Comment by ramya-rao-a** [comment]
Updated readme for Microsoft.Azure.EventHubs with a message matching the package description, but there was no readme for Microsoft.Azure.EventHubs.Processor to update

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj (hunk 1 above)
Comment (ramya-rao-a) [reply sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj:3]: Keeping the current text as concluded in https://github.com/Azure/azure-sdk-for-net/pull/20402#discussion_r614228896

File: sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj (hunk 2 above)
Comment (ramya-rao-a) [reply sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj:3]: Keeping current text as concluded in https://github.com/Azure/azure-sdk-for-net/pull/20402#discussion_r614228896

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 4.4.0-beta.1 (Unreleased)
+## 4.3.2 (2021-04-15)
```
Comment (ramya-rao-a) [inline sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md:3]:
```suggestion
## 4.3.2 (2021-04-26)
```

**Review by serkantkaraca — APPROVED** [review]
:shipit:

**Pull request merged**